src/window.ts

```diff
@@ -98,6 +98,20 @@ export class Window extends Adw.ApplicationWindow {
     this.default_width = note.width;
     this.default_height = note.height;
 
+    this.note.bind_property_full(
+      "title",
+      this,
+      "title",
+      GObject.BindingFlags.SYNC_CREATE,
+      (_, title) => {
+        if (!title) {
+          return [true, "Untitled Note"];
```

Comment:
I feel like "Sticky Note" may be better here.

Also, wrap the string in a `_("string")` so that it may get picked up by gettext (the translation infrastructure).

```suggestion
          return [true, _("Sticky Note")];
```

Reply:
done !